src/observations/radec_mpc.jl

```diff
@@ -0,0 +1,497 @@
+abstract type AbstractObservation end
+
```

Comment:
A related point: this is mostly conceptual if you will, but if it's not too big of an effort to change it, I would like to suggest the name `AbstractAstrometry` rather than `AbstractObservation`, just because we're dealing only with astrometric observations, and we're currently not using imaging, photometry, or some other kind of observational data (which would probably require a different abstraction anyways)

Reply:
I've changed `AbstractObservation` to `AbstractAstrometry`.